templates/takeovers/_german_takeover_k8.html

```diff
@@ -1,4 +1,4 @@
-<div data-lang="{{ locale }}" lang="de" class="p-strip--image is-dark js-takeover {% if not default %}u-hide{% endif %}" style="background-color:#326DE6">
+<div data-lang="{{ locale }}" class="p-strip--image is-dark js-takeover" style="background-color:#326DE6">
```

Comment:
Why have you removed the `lang` here?

Reply:
I did this because I couldn't tell what purpose it was serving. I don't like the duplication between `data-lang` and `lang`, it seems redundant.

However, thinking about it, I think maybe it isn't redundant. The `lang` attribute semantically marks the content as having a primary language of e.g. German, which is worthwhile. It would be nice if we could use the `lang` attribute for everything, but the main thing preventing this, as far as I can see, is that `lang` doesn't support multiple languages, it is only intended to demark the *primary* language.

So I'll reinstate the `lang` attribute, but I'd like to propose that instead of `data-lang` we have `data-lang-extra`. The implication here is that we use the value of the `lang` attribute in JavaScript to determine the default language, and then we can *optionally* add extra languages.

(For takeovers that are intended for *all* languages, we should indicate this by simply removing the `lang` attribute).

Reply:
Actually, inspecting the existing takeovers, none of them have used multiple languages. So actually the `data-lang-extra` attribute could remain theoretical at this point, to be implemented in the future if we need it. And for now, we just use the existence or non-existence of the `lang` attribute.

Reply:
I've now implemented this. Please re-review.